idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/kettlereboiler.py

```diff
@@ -0,0 +1,367 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+IDAES Kettle Reboiler
+
+Author : Paul Akula
+"""
+
+# Import Pyomo libraries
+from pyomo.environ import Expression, Constraint, value, Var, units
+from pyomo.common.config import ConfigBlock, ConfigValue, Bool
+
+# Import IDAES cores
+from idaes.core import (ControlVolume0DBlock,
+                        declare_process_block_class,
+                        MaterialBalanceType,
+                        MomentumBalanceType,
+                        UnitModelBlockData,
+                        useDefault)
+from idaes.core.util.config import is_physical_parameter_block
+from idaes.core.util import get_solver
+import idaes.logger as idaeslog
+
+# Set up logger
+_log = idaeslog.getLogger(__name__)
+
+
+@declare_process_block_class("KettleReboiler")
+class KettleReboilerData(UnitModelBlockData):
+    """Kettle Reboiler Unit Model."""
+
+    CONFIG = UnitModelBlockData.CONFIG()
+
+    # Configuration template for Phase specific  arguments
+    _PhaseCONFIG = ConfigBlock()
+
+    CONFIG.declare("specify_heat_duty", ConfigValue(
```

Comment:
Is this required as a config argument? Normally you would just check if the variable is fixed. Doing things this way implies that this is an upfront decision that cannot be changed later.